Marlin/src/module/stepper.cpp

```diff
@@ -1944,13 +1978,9 @@ uint32_t Stepper::stepper_block_phase_isr() {
 
       // For minimum pulse time wait before looping
       // Just wait for the requested pulse duration
-      if (LA_steps) {
-        while (HAL_timer_get_count(PULSE_TIMER_NUM) < pulse_end) { /* nada */ }
-        #if MINIMUM_STEPPER_PULSE
-          // Add to the value, the time that the pulse must be active (to be used on the next loop)
-          pulse_end += hal_timer_t(MIN_PULSE_TICKS);
-        #endif
-      }
```

Comment:
This exists to wait for a minimum duration that should apparently exist _between_ pulses. When using Linear Advance, an extra interrupt call is applied to handle the extruder. From my understanding this is a little bit of a hack, but it's possible that @ejtagle and other contributors have more knowledge about the full implications.